### PR TITLE
revert plateau pressure control, and simplify blower ramp-up

### PR DESCRIPTION
**Pull request by Benhalor**

### CODE REVIEW

File: src/software/firmware/srcs/screen.cpp [hunk 1]
```diff
@@ -78,8 +78,14 @@ void displayCurrentInformation(uint16_t peakPressure, uint16_t plateauPressure,
     screen.setCursor(0, 3);
     char message[SCREEN_LINE_LENGTH + 1];
 
-    (void)snprintf(message, SCREEN_LINE_LENGTH + 1, "%2u    %2u    %2u  meas", peakPressure / 10u,
+    // Case plateau is not detected
+    if (plateauPressure == UINT16_MAX){
+        (void)snprintf(message, SCREEN_LINE_LENGTH + 1, "%2u    ?    %2u  meas", peakPressure / 10u,
```
Comment (jabby) [inline src/software/firmware/srcs/screen.cpp:83]: I think there is a missing `space` or  `?` in this message. 
Line 86, pattern ` "%2u    %2u    %2u  meas"` has a length of 20 characters. (2 + 4 spaces + 2 + 4 spaces + 2 + 2 spaces + 4 letters).
Line 83 we have only 2 + 4 spaces + 1 + 4 spaces + 2 + 2 spaces + 4 letters = 19 characters.
I suggest to add an empty space before `?`.

File: src/software/firmware/srcs/pressure_controller.cpp [hunk 2]
```diff
@@ -132,15 +132,20 @@ void PressureController::initRespiratoryCycle() {
...
+    // Disable peak pressure control
+    /*else if (m_plateauPressure > (m_maxPlateauPressureCommand * 102u / 100u)) {
+        uint16_t plateauDiff = (((m_plateauPressure - m_maxPlateauPressureCommand) * 2u) / 10u);
+        onPeakPressureDecrease(min(plateauDiff, MAX_PEAK_INCREMENT));
+
+    // Increase peak pressure only if blower is stable.
+    } else if (m_plateauPressure < (m_maxPlateauPressureCommand * 98u / 100u) && m_blower_increment == 0u) {
+        uint16_t plateauDiff = (((m_maxPlateauPressureCommand - m_plateauPressure) * 2u) / 10u);
+        onPeakPressureIncrease(min(plateauDiff, MAX_PEAK_INCREMENT));
+    }*/
```
Comment (jabby) [inline src/software/firmware/srcs/pressure_controller.cpp:148]: If the revert is needed, we don't need to keep the removed code in comments. Git is here and save us from loosing interesting code.

File: src/software/firmware/srcs/pressure_controller.cpp [hunk 3]
```diff
@@ -280,7 +286,8 @@ void PressureController::onPlateauPressureDecrease() {
         m_maxPlateauPressureCommand = CONST_MIN_PLATEAU_PRESSURE;
     }
 
-    onPeakPressureDecrease();
+    // Disable plateau pressure control
+    //onPeakPressureDecrease();
```
Comment (jabby) [inline src/software/firmware/srcs/pressure_controller.cpp:290]: Same here about dead code.

File: src/software/firmware/srcs/pressure_controller.cpp [hunk 4]
```diff
@@ -292,7 +299,8 @@ void PressureController::onPlateauPressureIncrease() {
         m_maxPlateauPressureCommand = CONST_MAX_PLATEAU_PRESSURE;
     }
 
-    onPeakPressureIncrease();
+    // Disable plateau pressure control
+    //onPeakPressureIncrease();
```
Comment (jabby) [inline src/software/firmware/srcs/pressure_controller.cpp:303]: Same here about dead code.

File: src/software/firmware/srcs/screen.cpp (hunk 1 above)
Comment (dsferruzza) [reply src/software/firmware/srcs/screen.cpp]: Done! I caught it too ^^

File: src/software/firmware/srcs/pressure_controller.cpp (hunk 2 above)
Comment (dsferruzza) [reply src/software/firmware/srcs/pressure_controller.cpp]: And also done because it told myself the same!

**Pull request merged**